processor/deltatocumulativeprocessor/internal/lineartelemetry/metrics.go

```diff
@@ -16,12 +16,13 @@ import (
 )
 
 func New(set component.TelemetrySettings) (Metrics, error) {
+	zero := func() int { return -1 }
```

Comment:
Am I the only one who thinks "zero" is a bad name for something that returns `-1`? Also because the previous iteration of the code returned zero...

Reply:
ugh this is an oversight.
I will address this in a later PR